praw/exceptions.py

```diff
@@ -14,6 +14,14 @@
 _RedditErrorItem = TypeVar("_RedditErrorItem")
 
 
+class _Retry(Exception):
```

Comment:
Sorry I wasn't very specific earlier. I don't think we should have this retry logic included at all in this PR. Simply add the ability to one-time re-issue the request if a rate limit under the threshold is encountered.

Reply:
The problem with just re-trying is that the objector is called after the data is retrieved. There needs to be a way to signal to the code that we want to retry rather than raising an actual Exception.

Reply:
What I'm suggesting is make no changes here, but up outside of the request handle this specific APIException. We don't need to make a custom exception for this case.

Reply:
Maybe I can make a change to the underlying RedditAPIException code.

Reply:
I'm pretty sure @bboe is just suggesting you edit [the `reply` method](https://github.com/praw-dev/praw/blob/05962a7f9387788abbce7a0e5587c983055e0ee9/praw/models/reddit/mixins/replyable.py#L38-L41) to handle the case where you are replying to a chat post and you get a ratelimit error with a low enough value.

Reply:
@jarhill0 see slack.